docs/UserGuide.adoc

```diff
@@ -149,27 +160,37 @@ Returns `john` and `John Doe`
 * `find Betsy Tim John` +
 Returns any person having names `Betsy`, `Tim`, or `John`
 
-=== Deleting a person or a tag: `delete`
-==== Deleting a Person
+=== Deleting multiple persons or a tag: `delete`
+==== Deleting multiple persons
+
+Deletes the multiple specified persons from the address book. +
+Format: `delete [INDEX1], [INDEX2], [INDEX3]...` +
```

Comment:
What happens when `delete` is called without any parameters? This line implies that all parameters are optional

Reply:
oh i see!